### PR TITLE
New lines it_it.json

### PR DESCRIPTION
**Pull request by Alessio798**

### CODE REVIEW

**Comment by Alessio798** [comment]
Since `en_us.json` is not getting other changes, I'm undrafting this pr.

File: src/main/resources/assets/splatcraft/lang/it_it.json [hunk 1]
```diff
@@ -389,6 +393,46 @@
...
+  "gui.stage_pad.label.create_stage": "Crea scenario",
+  "gui.stage_pad.label.set_stage_name": "Nome:",
+  "gui.stage_pad.label.set_stage_name.textbox": "Scrivi il nome dello scenario...",
+  "gui.stage_pad.label.stage_id": "ID: %s",
+  "gui.stage_pad.label.corner_1": "Angolo 1:",
+  "gui.stage_pad.label.corner_2": "Angolo 2:",
+
+  "gui.stage_pad.button.create_stage": "+ Crea un nuovo scenario",
+  "gui.stage_pad.textbox.search_stage": "Scrivi il nome di uno scenario...",
+  "gui.stage_pad.button.search_stage": "Barra di ricerca on/*off",
```
Comment (Octol1ttle) [inline src/main/resources/assets/splatcraft/lang/it_it.json]: What is the asterisk for?

File: src/main/resources/assets/splatcraft/lang/it_it.json [hunk 2]
```diff
@@ -495,26 +539,34 @@
...
   "gamerule.splatcraft.inkableGround": "Le armi Splatcraft possono colorare terreno",
-  "gamerule.splatcraft.inkableGround.description": "Permette alle armi Splatcraft di colorare i blocchi ovunque",
+  "gamerule.splatcraft.inkableGround.description": "Permette alle armi Splatcraft di colorare i blocchi ovunque nel mondo",
   "gamerule.splatcraft.rechargeableInkTank": "Serbatoio ricaricabile",
-  "gamerule.splatcraft.rechargeableInkTank.description": "Permette ai serbatoi di ricarsi passivamente, più velocemente mentre si è immersi.",
+  "gamerule.splatcraft.rechargeableInkTank.description": "Permette ai serbatoi di ricarsi passivamente, più velocemente mentre si è immersi",
+  "gamerule.splatcraft.globalSuperJumping": "Super salto globale",
+  "gamerule.splatcraft.globalSuperJumping.description": "Permette ai giocatori di eseguire super salti fuori dagli scenari",
+  "gamerule.splatcraft.superJumpDistanceLimit": "Limite di distanza super salto",
+  "gamerule.splatcraft.superJumpDistanceLimit.description": "Determina la massima distanza di super salto al di fuori degli scenari. §oI valori negativi disabilitano interamente il limite",
```
Comment (Octol1ttle) [inline src/main/resources/assets/splatcraft/lang/it_it.json]: `§o` isn't present in English, is there any reason it's here?

File: src/main/resources/assets/splatcraft/lang/it_it.json (hunk 1 above)
Comment (Alessio798) [reply src/main/resources/assets/splatcraft/lang/it_it.json]: Oops! I'll fix it when possible

File: src/main/resources/assets/splatcraft/lang/it_it.json (hunk 2 above)
Comment (Alessio798) [reply src/main/resources/assets/splatcraft/lang/it_it.json]: `§o` just makes the text in italic; I'll just remove it.

**Review by Octol1ttle — APPROVED** [review]

**Pull request merged**